### PR TITLE
scrabble-score: convert to instructions and introduction

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

**Review by IsaacG — CHANGES_REQUESTED** [review]

File: exercises/scrabble-score/introduction.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Introduction
+
+[Scrabble][wikipedia] is a word game where players place letter tiles on a board to form words. Each letter has a value. A word's score is the sum of its letter values.
```
Comment (MatthijsBlom) [inline exercises/scrabble-score/introduction.md:3]: Nitpick:

```suggestion
[Scrabble][wikipedia] is a word game where players place letter tiles on a board to form words. Each letter has a value. A word's score is the sum of its letters' values.
```

File: exercises/scrabble-score/instructions.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+```text
+Letter                           Value
+A, E, I, O, U, L, N, R, S, T     1
+D, G                             2
+B, C, M, P                       3
+F, H, V, W, Y                    4
+K                                5
+J, X                             8
+Q, Z                             10
+```
```
Comment (MatthijsBlom) [inline exercises/scrabble-score/instructions.md:18]: This _could_ be a table instead of a code block.

**Review by MatthijsBlom — APPROVED** [review]

**Review by IsaacG — APPROVED** [review]

File: exercises/scrabble-score/instructions.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Instructions
+
+Compute a word's Scrabble score by summing its letters' values.
+
+## Letter Values
+
```
Comment (iHiD) [inline exercises/scrabble-score/instructions.md:6]:
```suggestion
```

File: exercises/scrabble-score/instructions.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+| D, G                         | 2     |
+| B, C, M, P                   | 3     |
+| F, H, V, W, Y                | 4     |
+| K                            | 5     |
+| J, X                         | 8     |
+| Q, Z                         | 10    |
+
+## Examples
+
+The word "cabbage" is worth 14 points:
```
Comment (iHiD) [inline exercises/scrabble-score/instructions.md:21]:
```suggestion
For example, the word "cabbage" is worth 14 points:
```

File: exercises/scrabble-score/instructions.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
+# Instructions
+
+Compute a word's Scrabble score by summing its letters' values.
```
Comment (iHiD) [inline exercises/scrabble-score/instructions.md:3]:
```suggestion
Your task is to compute a word's Scrabble score by summing the values of its letters.
```

I suggest this slight variation so we don't have the same phrase "summing it's letter's values" twice in a row.

**Comment by ErikSchierboom** [comment]
Thanks everybody! I merged this as we need it for our video :)

**Pull request merged**